core/modules/views_ui/js/views-admin.js

```diff
@@ -197,7 +197,7 @@ Backdrop.viewsUi.AddItemForm = function ($form) {
 
 Backdrop.viewsUi.AddItemForm.prototype.handleCheck = function (event) {
   var $target = $(event.target);
-  var label = $.trim($target.next().text());
+  var label = $.trim($target.closest("td").next().text());
```

Comment:
I beliebe that we prefer single to double quotes unless there's a specific reason:
```suggestion
  var label = $.trim($target.closest('td').next().text());
```